### PR TITLE
Removed root requirement

### PR DESCRIPTION
**Pull request by codrcodz**
- Updated makefile to temporarily allow for bashellite user to write to /etc/bashellite
- Updated metadata dir to /etc/bashellite
- Removed EUID check
- Moved bin dir into /home/bashellite temporarily
- All changes are in preparation for redesign of conf/metadata system

### CODE REVIEW

File: bashellite.sh [hunk 1]
```diff
@@ -372,25 +348,25 @@ Validate_repo_metadata() {
...
-    Warn " aptmirror => ${script_dir}/_metadata/${repo_name}/aptmirror_url.conf";
+    Warn " rsync => ${metadata_tld}/${repo_name}/rsync_url.conf";
+    Warn " http => ${metadata_tld}/${repo_name}/http_url.conf";
+    Warn " pypi => ${metadata_tld}/${repo_name}/pypi_url.conf";
+    Warn " aptmirror => ${metadata_tld}/${repo_name}/aptmirror_url.conf";
     Warn "Please place any desired filter parameters, (aka includes and/or excludes), in:";
-    Warn " => ${script_dir}/_metadata/${repo_name}/repo_filter.conf";
-    Fail "Place the URL(s) of the repo (${repo_name}) in one, and ONLY one, \"_metadata/*_url.conf\" file and try again.";
+    Warn " => ${metadata_tld}/${repo_name}/repo_filter.conf";
+    Fail "Place the URL(s) of the repo (${repo_name}) in one, and ONLY one, \"${script_name}/${repo_name}/<provider>_url.conf\" file and try again.";
```
Comment (ericlake) [inline bashellite.sh]: I don't see where script_name is being defined.

File: bashellite.sh [hunk 2]
```diff
@@ -214,6 +208,7 @@ Parse_parameters() {
   unset repo_name;
   unset dryrun;
   unset all_repos;
+  unset metadata_tld
```
Comment (ericlake) [inline bashellite.sh]: Add ';'

File: bashellite.sh (hunk 1 above)
Comment (codrcodz) [reply bashellite.sh]: I stripped this var out, and must have missed this reference. I will fix and submit another commit.

File: bashellite.sh (hunk 2 above)
Comment (codrcodz) [reply bashellite.sh]: good catch.

**Pull request closed without merging**